styleguide/source/_patterns/01-molecules/06-components/10-hub-hero/hub-hero-no-overlay.twig

```diff
@@ -0,0 +1,14 @@
+{#
+ # @file hub-hero.twig
+ # Copyright 2017 Palantir.net, Inc.
+ #}
+
+<section class="hub-hero hub-hero-no-overlay hub-hero-image-123">
```

Comment:
As discussed, please move the background-image declaration into an inline style and remove this class.